mobile/cafes_test.go

```diff
@@ -254,6 +254,14 @@ func flushCafeRequests(limit int) (int, error) {
 		}
 	}
 
+	// make sure we can query for each
+	for g := range reqs {
+		bytes, err = cafesTestVars.mobile.CafeRequest(g)
```

Comment:
since you haven't created a bytes yet, you'll need to use `bytes, err := cafesTest...`